modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/Settings.kt

```diff
@@ -329,6 +329,7 @@ interface Settings {
     val autoDownloadUpNext: UserSetting<Boolean>
 
     val useEmbeddedArtwork: UserSetting<Boolean>
+    val useEpisodeArtwork: UserSetting<Boolean>
```

Comment:
Can you change the name of this property and all others where you reference it as `episodeArtwork` to `rssArtwork`? This will be a bit clearer and be consistent with the setting translation.